src/devices/Mcp3xxx/Mcp3001.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Spi;
+
+namespace Iot.Device.Adc
+{
+    /// <summary>
+    /// MCP3001 Analog to Digital Converter (ADC)
+    /// </summary>
+    public class Mcp3001 : Mcp3Base
+    {
+        /// <summary>
+        /// Constructs Mcp3008 instance
+        /// </summary>
+        /// <param name="spiDevice">Device used for SPI communication</param>
+        public Mcp3001(SpiDevice spiDevice) : base(spiDevice) { }
+
+        /// <summary>
+        /// Reads a 10-bit (0..1023) value from the device
+        /// </summary>
+        /// <returns>10-bit value corresponding to relative voltage level on specified device channel</returns>
+        public int Read()
+        {
+            return ReadInternal(adcRequest: 0, adcRequestLengthBytes: 2, 10 + 3, delayBits: 0) >> 3;
```

Comment:
explicit arg name for `10+3` and please add a comment why reading 13 bits for 10-bit ADC (unless the summary comment is wrong)

Reply:
The answer to this is that all the xxx1 devices don’t have a start bit being sent so the readout of the data is a certain number of bits after the /cs pin goes low. This means that unlike the xxx8, xxx4 and xxx2 devices where the b0 result is aligned with the b0 of the returned int the result data is potentially somewhere in the middle and has to be shifted. I have documented. The bit patterns in the base class.

Reply:
sounds good, just double checking this is not accidental